recipe_scrapers/norecipes.py

```diff
@@ -10,7 +10,7 @@ def host(cls):
         return "norecipes.com"
 
     def author(self):
-        return self.soup.find("meta", {"name": "author"}).get("content")
+        return self.schema.author()
```

Comment:
Nice :)  :tada:

Reply:
Oh, I'm viewing the wrong browser tab and complimenting myself by accident.  I thought this was a different PR..